### PR TITLE
Add link to submission form

### PR DESCRIPTION
**Pull request by samgoldman**
This is an attempt to address #101 
Bot repository: https://github.com/2020PB-bot/police-brutality
Website repository: https://github.com/samgoldman/police-brutality-submissions

TODO before before taking out of draft:
- [x] Test with bogus incident: https://github.com/2020PB/police-brutality/issues/414
- [ ] Test with real incident
    - [ ] Test & define merge process
- [x] Add support for incident IDs
- [x] Add notice to read contribution guidelines
- [x] Add incidents to the end of the city list

### CODE REVIEW

**Comment by samgoldman** [comment]
@2020PB @maxspencer @00000100-00010100 @andrewsnyder328
I'm pretty sure I'm done with this version of the tool. If someone could test it with a real incident, I would appreciate it - I want to make sure it's straightforward to use and merge. 
I've added @2020PB and @00000100-00010100 to the bot's fork so you guys can create PRs.

**Comment by samgoldman** [comment]
I've taken this out of draft mode. I haven't yet tested on a real incident, but I've done several local tests, fixed some things, and am fairly confident that it's ready for other people to use. Let me know if you have any questions or concerns.

**Comment by adjkant** [comment]
Thanks again for your work on this!

One small nit, but I think it will get more attention and look both more noticeable and professional if it is a linked text rather than a full Heroku URL + on a separate line. Something like:

**Don't want to create a Github account?**
[Fill out this anonymous reporting form.](https://police-brutality-submissions.herokuapp.com/)

I also found the load a bit slow, which I have seen in my experience with the Heroku free tier which can encounter a lot of cold starts if not hit frequently. if we could upgrade the Heroku hosting level I would be happy to personally fund that for a few months so long as reports are coming in ([Hobby will prevent "sleeping" and should be sufficient but we could always overdo it to be safe with Standard](https://www.heroku.com/pricing)). I'd hate to lose reports thanks to people thinking the link was dead.

**Comment by samgoldman** [comment]
@adjkant Good suggestion, made that change. 

And yeah, the speed issue is probably because of Heroku shutting it down. One option might be to have a script that queries the page periodically to prevent shutdown.

**Comment by amardeshbd** [comment]
This is great! I was using the GitHub issue [submission link](https://github.com/amardeshbd/android-police-brutality-incidents/blob/develop/android-app/app/src/main/java/com/blacklivesmatter/policebrutality/config/Constants.kt#L17) in the Android app I am building. 

So, I believe I can safely replace that with the new Heroku URL? Since it's running off of Heroku free tier, it's never supposed to go offline right @samgoldman ?

**Comment by samgoldman** [comment]
@amardeshbd The submission form hasn't actually been approved yet, but you can link to it once it is. It won't go offline, but it might have to cold start.
@00000100-00010100 @2020PB @maxspencer Any thoughts on approving the submission form?

File: README.md [hunk 1]
```diff
@@ -8,6 +8,9 @@
 👉🏿 **[Report it here!](https://github.com/2020PB/police-brutality/issues/new?assignees=&labels=Incident+report&template=incident-report.md&title=Incident+in+CITY%2C+STATE)** 👈🏻<br/>
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;👆🏼&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
 
+**Don't want to create a Github account?**
+[Fill out this anonymous reporting form.](https://https://police-brutality-submissions.herokuapp.com/)
```
Comment (ubershmekel) [inline README.md:12]: https://https://

Comment (samgoldman) [reply README.md]: Fixed

**Comment by ubershmekel** [comment]
Still seems broken :(

I clicked submit, I didn't see a spinner, it's as though nothing happened. I clicked submit again, and nothing happened, then after ~10 seconds the CAPTCHA said it was invalid, I re-captcha-ed, I clicked submit again, nothing happened, then it said the CAPTCHA was invalid again.

image

...

image

**Comment by samgoldman** [comment]
@ubershmekel My bad - I don't have the most experience with UI. It's doing basic form validation on the date and has disabled it due to it not matching. I just pushed an update that highlights the text box if it's not valid.

**Pull request closed without merging**